lib/apis/v3/bgpconfig.go

```diff
@@ -67,6 +76,26 @@ type ServiceClusterIPBlock struct {
 	CIDR string `json:"cidr,omitempty" validate:"omitempty,net"`
 }
 
+// Community contains community value and its name.
+// BGP community value can be of format `aa:nn` or `aa:nn:mm`.
```

Comment:
This is misplaced - comment describing the syntax for a community should be on the `Value` section, not on the community struct.